website/templates/docs.html

```diff
@@ -517,11 +518,14 @@ <h2 id=option>Option types</h2>
 
 fn main() {
 	repo := <comment>// ... initialize the repository here and load users data </comment> 
+	// either get the user or handle an error and leave the current scope
 	user := repo.find_user_by_id(10) or {
 		eprintln(err) 
 		return 
 	} 
 	println(user.id) // 10 
+	user = repo.find_user_by_id(11)? <comment>// ? propagates an error </comment>
+	println(user.id) // 11
```

Comment:
I added these lines so the `statement?` error propagation is documented under the Option types section.